examples/psyad/lfric/adjoint/lbl_adj_poly2d_reconstruction_kernel_mod.F90

```diff
@@ -0,0 +1,73 @@
+module adj_poly2d_reconstruction_lbl_kernel_mod
```

Comment:
No, there's no convention here. 'lbl' stands for "line-by-line" and means that the code has been adjointed on a 'line-by-line' basis so that it is a correct adjoint but it is *not* a valid LFRic kernel because we end up with writes to columns other than the one passed to the kernel. This is therefore only a step along the way to producing a valid, adjointed LFRic kernel. As such, it's not something we have a convention for as, ultimately, it isn't what we want to be creating.